### PR TITLE
Draft PR: L2 Transaction Data Fee Specification

### PR DESCRIPTION
**Pull request by Therecanbeonlyone1969**

### CODE REVIEW

File: workitems/l2-transaction-data-fee/l2-transaction-data-fee-v1.0-psd01.md [hunk 1]
```diff
@@ -196,55 +264,120 @@ Note that requirements are uniquely numbered in ascending order within each requ
...
-# 3 Layer 2 Transaction Fees Specification
+# 3 Layer 2 Transaction Data Fee Specification
+###### L2TFSPECSEC
 
-TBD
+#### **[R1]**
+An L2 Transaction Data Fee MUST be indicated to be part of the Execution Fee.
+
+#### **[R2]**
+An L2 Transaction Data Fee MUST be displayed separately from the Execution Fee.
```
Comment (smartcontracts) [inline workitems/l2-transaction-data-fee/l2-transaction-data-fee-v1.0-psd01.md:313]: Conflicting with R1?

**Comment by smartcontracts** [comment]
I still think we should use the terminology that the "L2 transaction fee = execution fee + data fee"

File: workitems/l2-transaction-data-fee/l2-transaction-data-fee-v1.0-psd01.md (hunk 1 above)
Comment (Therecanbeonlyone1969) [reply workitems/l2-transaction-data-fee/l2-transaction-data-fee-v1.0-psd01.md:313]: @smartcontracts not really ... you can list the execution fee something like this `(EF: 10, DF: 6, Rest: 4)` that meets R1 and R2

**Comment by Therecanbeonlyone1969** [comment]
> I still think we should use the terminology that the "L2 transaction fee = execution fee + data fee"

We can talk about that during the next WG meeting. Not against it @smartcontracts

**Pull request closed without merging**